packages/fuels-core/src/code_gen/abigen.rs

```diff
@@ -54,6 +57,11 @@ impl Abigen {
         self
     }
 
+    pub fn script(mut self) -> Self {
```

Comment:
We could remove this with the bellow alternative.

Reply:
No it's actually needed for other methods that check the value of the `is_script` boolean. Especially if we try to factorize things down further.